src/test/java/org/opensearch/jobscheduler/JobSchedulerPluginRestIT.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.jobscheduler;
+
+import org.junit.Assert;
+import org.opensearch.client.Request;
+import org.opensearch.client.Response;
+import org.opensearch.common.xcontent.LoggingDeprecationHandler;
+import org.opensearch.common.xcontent.NamedXContentRegistry;
+import org.opensearch.common.xcontent.json.JsonXContent;
+import org.opensearch.test.rest.OpenSearchRestTestCase;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+public class JobSchedulerPluginRestIT extends OpenSearchRestTestCase {
+
+    public void testPluginsAreInstalled() throws IOException {
+        Request request = new Request("GET", "/_cluster/health");
```

Comment:
I know this was in the original test, but do you know why we verify that the cluster is green?

Reply:
hi @downsrob , i think this will test that after loading job-scheduler is OS cluster in green health state or not.

Reply:
If you think there are any issues which verifying the cluster health here could catch, and that there aren't any cases in a remote testing scenario where this would fail when there actually isn't an issue then I think it is fine to leave it.